src/XliffTasks/build/XliffTasks.targets

```diff
@@ -109,7 +97,19 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="GetXlfSources" Returns="@(XlfSource)" />
+  <Target Name="GetXlfSources" Returns="@(XlfSource)">
```

Comment:
I think there may now be missing dependencies on this. Still studying.

Reply:
What dependencies might be missed?  I chose to move the gathering of `@(XlfSource)` into this target specifically because that was this target's return value.

Reply:
We now need to make sure that anything that uses @(XlfSource) depends on GetXlfSources.

Studying it now, I think there is just one case where this doesn't already hold: GatherXlf now needs a dependency on GetXlfSources. Currently, it depends only on EnsureXlfIsUpToDate -> _UpdateXlf -> GetXlfSources, but that is conditioned on ErrorOnOutOfDateXlf being true.

The fix would be 

``` diff
<Target Name="GatherXlf" 
-        DependsOnTargets="EnsureXlfIsUpToDate"
+        DependsOnTargets="GetXlfSources;EnsureXlfIsUpToDate"
```

Reply:
There are two targets that consume `@(XlfSource)` and they all have `GetXlfSources` in their dependency chain:

1. `UpdateXlfSourceList` consumes `@(XlfSource)`, but the target `_UpdateXlf` depends on `GetXlfSources;UpdateXlfSourceList` so presumably `GetXlfSources` will always be executed before `UpdateXlfSourceList`?
2. `GatherXlf` depends on `EnsureXlfIsUpToDate` which depends on `_UpdateXlf` which depends on `GetXlfSources`.

So it appears that all dependencies are already met, but if you prefer we can add the explicit dependency on `GetXlfSources` from both locations, `GatherXlf` like you mentioned and `UpdateXlfSourceList`.